### PR TITLE
recover all info from CONAN database

### LINKED ISSUE #145: Recover the textual information for SCCS v1716-v1780
@bibiko @xrotwang @SimonGreenhill  - are you able to extract data from a dBase database? See below.

"A disk included with this volume holds zipped files for the CONAN data base and a copy of a program needed to unzip the files. The CONAN date base requires dBASE IV (or later) to run and the files must be unzipped to a hard drive.  The numeric codes in ST81.DAT and ST82.DAT duplicate to codes in CONAN, **but you cannot recover the textual information in coding decisions for specific cases without running the dBASE program.**"

"The CONAN materials on the disk should all be copied to directory on
your hard disk. If you have a directory dedicated to dBASE files you might
want to copy it to that directory.  Unzip the compressed files by issuing
the commands: "pkunzip conan" and "pkunzip concodes".  To use the CONAN
materials start dBASE and open the CONAN database."

Here's a link to the zipped 'conan' and 'concodes' files: 
https://www.dropbox.com/s/654xp89abk8tpm0/CONAN.ZIP?dl=0
https://www.dropbox.com/s/qy624qlmwjqgvr6/CONCODES.ZIP?dl=0

Text file describing the files (conan includes 67 codes for a 101 society subsample of the SCCS):
https://www.dropbox.com/s/7o7v4scv7h7fjj6/Conan.txt?dl=0

And the MAPTAB files (from which I've already extracted the variables and codes - they do not contain textual info):
https://www.dropbox.com/s/kxtgm6wbechcbns/STDS81_82_CONAN_MAPTAB.zip?dl=0

### PR DESCRIPTION
**Pull request by xrotwang**
closes #145

### CODE REVIEW

File: datasets/SCCS/codes.csv [hunk 1]
```diff
@@ -9023,409 +9023,409 @@ SCCS1713,NA,Missing data,NA
...
-SCCS1725,3,access to land of good quality (original code 21),access to land of good quality
-SCCS1725,4,access to land of restricted quality (original code 22),access to land of restricted quality
-SCCS1726,NA,Missing data,NA
+SCCS1721,NA,missing data,missing data
+SCCS1721,10,absence of rich,absence of rich
+SCCS1721,20,"presence of rich, no information on numbers","presence of rich, no information on numbers"
+SCCS1721,21,few rich,few rich
+SCCS1721,22,many rich,many rich
+SCCS1722,NA,missing data,missing data
+SCCS1722,88,absence of rich (variable 11 coded ´10´),absence of rich (variable 11 coded ´10´)
```
Comment (xrotwang) [inline datasets/SCCS/codes.csv]: @kirbykat I realize I have to adapt the references to other SCCS variables in the descriptions here.

File: datasets/sources.bib [hunk 2]
```diff
@@ -36467,3 +36467,799 @@ @article{zwernemann1963
...
+    bibkey = {}
+}
+@book{-184,
+    author = {M},
+    bibkey = {}
+}
+@book{-257,
+    author = {Bu},
+    bibkey = {}
+}
```
Comment (xrotwang) [inline datasets/sources.bib]: @kirbykat These first three records are artifacts, due to unparseable comment fields and must be removed. Will do so later today.

**Comment by xrotwang** [comment]
There are still some things to clean up:
- [ ] the new sources need checking and completing the BibTeX
- [ ] many of the comments parsed from CONAN's `TEXT_COM` field contain trailing garbage, which should be removed

**Comment by kirbykat** [comment]
@xrotwang this looks fabulous! thank you! i am happy to merge, but not sure if you wanted to complete the final things mentioned above before doing so. ...I'll let you hit merge when you think it is ready.

**Comment by xrotwang** [comment]
@kirbykat I'll fix the outstanding issues and then merge.

**Pull request merged**